main/src/cgeo/geocaching/SearchActivity.java

```diff
@@ -123,6 +133,17 @@ public final void onCreate(final Bundle savedInstanceState) {
         init();
     }
 
+    private static void removeFlags(final Intent intent, final int flags) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
Isn't this IF statement doubled with the calling part above?

Reply:
Forgot to delete unused method, will remove it (codacy complains too...)